package.json

```diff
@@ -89,6 +91,8 @@
         "raw-loader": "^3.1.0",
         "react": "^16.7.0",
         "react-dom": "^16.7.0",
+        "react-intl": "^2.9.0",
+        "react-tether": "^1.0.0",
```

Comment:
> Where is react-tether used in the virtualized table?
If I don't add it, the building process gives me error about dropdown-menu. I guess it's because one of the renderers uses breadcrumb and breadcrumb imports dropdown-menu.

Reply:
> Where is react-tether used in the virtualized table?

If I don't add it, the building process gives me error about dropdown-menu. I guess it's because one of the renderers uses breadcrumb and breadcrumb imports dropdown-menu.